resources/istio/values.yaml

```diff
@@ -125,8 +139,12 @@ pilot:
     requests:
       cpu: 100m
       memory: 128Mi
+  rollingMaxSurge: 1
+  rollingMaxUnavailable: 0
   env:
     PILOT_HTTP10: 1
+    PILOT_SKIP_VALIDATE_TRUST_DOMAIN: "true"
+  enableProtocolSniffing: false
```

Comment:
that value is not used anywhere in the chars. Probably should be `enableProtocolSniffingForOutbound` and `enableProtocolSniffingForInbound` instead